chemprop/models/mpn.py

```diff
@@ -76,6 +84,10 @@ def forward(self,
             if self.features_only:
                 return features_batch
 
+        if atom_descriptors_batch is not None:
+            atom_descriptors_batch = [np.zeros([1, atom_descriptors_batch[0].shape[1]])] + atom_descriptors_batch   # padding the first with 0 to match the atom_hiddens
+            atom_descriptors_batch = torch.from_numpy(np.concatenate(atom_descriptors_batch, axis=0)).float().to(self.device)
+
```

Comment:
The `atom_descriptors_batch` isn't actually used until line 134, right? Do you think you could move this code inside the `if` statement on line 134 so that it's a little easier to see all the related code together?